app/styles/_overview.less

```diff
@@ -358,7 +358,7 @@
       color: #4d5258;
     }
   }
-  .list-view-pf-actions .actions-dropdown-kebab {
+  .list-pf-actions .actions-dropdown-kebab {
```

Comment:
There's a `.list-pf-actions` rule above we should combine this with.

I'm a little confused why this wasn't a problem in 3.7, though.